visualization/app/codeCharta/ui/areaSettingsPanel/areaSettingsPanel.component.html

```diff
@@ -30,9 +29,9 @@
 		class="md-primary"
 		ng-model="$ctrl._viewModel.dynamicMargin"
 		ng-change="$ctrl.applyDynamicMargin()"
-		title="Use a fit to width margin to see the whole map"
+		title="Set margin to default (50)."
 	>
-		Dynamic Margin
+		Default margin (50)
```

Comment:
```suggestion
		Default margin (50px)
```
Should we show the unit like this?